### PR TITLE
Further UWSGI changes for current live issue

### PR DESCRIPTION
**Pull request by Pheonnexx**
## What does this pull request do?

added alarm-backlog - to trigger a reset when the queue is full
added harakiri settings -  to stop workers taking too long.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: docker/cla_frontend.ini [hunk 1]
```diff
@@ -12,3 +12,5 @@ buffer-size=32768
 post-buffering-bufsize=32768
 http-workers=5
 alarm=removefile cmd:rm /tmp/listen_queue_healthy
+alarm-backlog=removefile cmd:rm /tmp/listen_queue_healthy
```
Comment (said-moj) [inline docker/cla_frontend.ini:15]: I think we just need the alarm name here alarm-backlog

File: docker/cla_frontend.ini [hunk 2]
```diff
@@ -12,3 +12,5 @@ buffer-size=32768
...
+harakiri=20
```
Comment (jogilder) [inline docker/cla_frontend.ini:16]: did we not want harakiri only if the alarm was triggered?

Comment (said-moj) [reply docker/cla_frontend.ini:16]: @jogilder is it possible to tie `harakiri` to alarm?
My understand is that `harakiri` accepts a value in seconds and any requests that takes longer that value will be dropped and the worker recycled https://uwsgi-docs.readthedocs.io/en/latest/Glossary.html

File: docker/cla_frontend.ini (hunk 1 above)
Comment (Pheonnexx) [reply docker/cla_frontend.ini:15]: amended

**Review by said-moj — APPROVED** [review]

**Pull request merged**